bootstrap.sh

```diff
@@ -16,4 +16,4 @@ sudo -u postgres psql -c "ALTER ROLE postgres WITH PASSWORD 'givit';"
 
 # run the server in background
 pipenv run python3 manage.py migrate
-nohup pipenv run python3 manage.py runserver 0:8000 &
+nohup pipenv run python3 manage.py runserver --noreload 0:8000 &
```

Comment:
i red somewhere that this needed but i try now and so that its work without this so i erase the --noreload and update the coomit,  no changes in bootstrap.sh.